Source/PluginEditor.h

```diff
@@ -10,11 +10,28 @@
 
 #include <JuceHeader.h>
 #include "PluginProcessor.h"
+#include "APSliderLook.h"
+#include "CustomSlider.h"
+#include "MixerButton.h"
+#include "ShowHideContainer.h"
+#include "APDefines.h"
 
 //==============================================================================
 /**
 */
-class Ap_dynamicsAudioProcessorEditor  : public juce::AudioProcessorEditor
+const static int M_HEIGHT = 500;
+const static int M_WIDTH = 700;
+const static int SLIDER_Y = 290;
+const static int SLIDER_WIDTH = 200;
+const static float FONT_HEIGHT = 24.0f;
+const static float SHADOW_FONT_HEIGHT = 28.0f;
+
+const static juce::Colour SHADOW_COLOR = juce::Colours::black.withAlpha(0.2f);
+const static juce::Colour INNER_GRADIENT_BG = juce::Colour(0xFFFFDC93);
+const static juce::Colour OUTER_GRADIENT_BG = juce::Colour(0xFFFFC446);
```

Comment:
If these are only used in the cpp, and not header, then use an anonymous namespace in the .cpp file. No need to be static.
If they are in the header, you will get extra re-compiles when you tweak the values, not so in the .cpp